crates/re_types/definitions/rerun/components/indices.fbs

```diff
@@ -0,0 +1,21 @@
+include "arrow/attributes.fbs";
+include "python/attributes.fbs";
+include "rust/attributes.fbs";
+
+include "rerun/datatypes.fbs";
+include "rerun/attributes.fbs";
+
+namespace rerun.components;
+
+// ---
+
+/// A generic index into a container.
+struct Index (
```

Comment:
Is this even used?

Reply:
No, I was pretty sure I got rid of that stuff